db/schema.rb

```diff
@@ -25,7 +25,7 @@
   add_index "relationships", ["follower_id"], name: "index_relationships_on_follower_id"
 
   create_table "tweets", force: :cascade do |t|
-    t.string   "content",    limit: 255
+    t.string   "content"
```

Comment:
__[rubocop]__
* Put one space between the method name and the first argument.
* Prefer single-quoted strings when you don't need string interpolation or special symbols.